build.sbt

```diff
@@ -74,7 +68,13 @@ lazy val microservice = Project(appName, file("."))
     Test / parallelExecution := false
   )
   .settings(majorVersion := 0)
-  .settings(scalacOptions ++= Seq("-Ypartial-unification"))
+  .settings(
+    scalacOptions ++= Seq(
+    "-Wconf:cat=unused&src=.*RoutesPrefix\\.scala:s",
+    "-Wconf:cat=unused&src=.*Routes\\.scala:s",
+    "-Wconf:cat=unused&src=.*ReverseRoutes\\.scala:s"
+    )
+  )
 
 lazy val acceptanceTestSettings =
```

Comment:
Can we add a section

`  commands += Command.command("testAll") { state =>
      "test" :: "it:test" :: state
  }
  `

Reply:
I think this needs test :: it:test :: acceptance:test if this has all 3

Reply:
It just needed 2  test and acceptanceTest